semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Settings.scala

```diff
@@ -1,31 +1,56 @@
 package scala.meta.internal.metacp
 
+import java.io.PrintStream
+
+import org.langmeta.io.AbsolutePath
+import org.langmeta.io.Classpath
+
+import scala.meta.cli.Metacp
+
 final case class Settings(
-    cps: List[String] = Nil,
-    d: String = "."
-)
+    out: PrintStream = System.out,
+    err: PrintStream = System.err,
+    classpath: Classpath = Classpath(Nil),
+    d: AbsolutePath = Metacp.defaultCacheDir,
+    directories: DirectoryOutput = DirectoryOutput.TempDirectory
+) {
+  def withClasspath(e: String) =
```

Comment:
`withX` usually means "take existing value of X and replace it with the argument". Let's call this method `appendX` instead?

Reply:
I agree that `withX` is not appropriate here, appendX would be better.

Reply:
No longer relevant.